components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java

```diff
@@ -245,8 +303,11 @@ private boolean initModels() {
                     WSDLParamDefinition wsdlParamDefinition = new WSDLParamDefinition();
                     ModelImpl model = new ModelImpl();
                     Property currentProperty = null;
-                    traverseTypeElement(node, null, model, currentProperty);
-                    if (StringUtils.isNotBlank(model.getName())) {
+                    try {
+                        traverseTypeElement(node, null, model, currentProperty);
+                    } catch (APIManagementException e) {
+                        throw new APIMgtWSDLException(e);
+                    }                    if (StringUtils.isNotBlank(model.getName())) {
```

Comment:
Can you fix the formatting issue?

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10668/commits/e39d2b2ac4948ce899f5c94672903c727ef5aae3